cfgov/research_reports/models.py

```diff
@@ -40,10 +40,19 @@ def sec(request, page):
             'url': format('#section-{}', i),
             'numbering': format('{}: ' if is_appendix else '{}. ', i),
             'children': [{
-                'title': subsection.sub_header,
-                'body': subsection.sub_body,
+                'title': subsection.header,
+                'body': subsection.body,
                 'url': format('#section-{}.{}', i, j),
-                'numbering': '' if is_appendix else format('{}.{} ', i, j)
+                'numbering': '' if is_appendix else format('{}.{} ', i, j),
+                'children': [{
+                    'title': section3.header,
+                    'body': section3.body,
+                    'url': format('#section-{}.{}.{}', i, j, k),
+                    'numbering': '' if is_appendix else
```

Comment:
Are Appendix headers just not going to be numbered, or will we be adding that later?

Reply:
Right, they won't be numbered (for subheaders and header-level-3)